hack/docs/metrics_gen/main.go

```diff
@@ -39,6 +40,16 @@ type metricInfo struct {
 	help      string
 }
 
+var (
+	stableMetrics = []string{"controller_runtime", "aws_sdk_go", "client_go", "leader_election", "interruption", "cluster_state", "workqueue", "karpenter_build_info", "karpenter_nodepool_usage", "karpenter_nodepool_limit",
```

Comment:
This set of metrics is going to quickly become out-of-date when we introduce other metrics or change names and folks are going to forget to update this doc. Do we have a plan for how we are going to ensure that folks make the right changes as things are merged in?